lib/ansible/module_utils/kubevirt.py

```diff
@@ -0,0 +1,179 @@
+# -*- coding: utf-8 -*-
+#
+
+# Copyright (c) 2018, KubeVirt Team <@kubevirt>
+# Apache License, Version 2.0
+# (see LICENSE or http://www.apache.org/licenses/LICENSE-2.0)
+
+from collections import defaultdict
+
+from ansible.module_utils.k8s.raw import KubernetesRawModule
+
+from openshift import watch
+from openshift.helper.exceptions import KubernetesException
+
+API_VERSION = 'kubevirt.io/v1alpha2'
+
+
+VM_COMMON_ARG_SPEC = {
+    'merge_type': {'type': 'list', 'choices': ['json', 'merge', 'strategic-merge']},
+    'wait': {'type': 'bool', 'default': True},
+    'wait_time': {'type': 'int', 'default': 30},
```

Comment:
… especially with a wait_time this low. On my test setup I regularly hit a timeout because of this. The default `wait_timeout` is 120 in `k8s`.

Also, it's called `wait_timeout` in `k8s` now (added in 2.8). We should use the same naming.

Reply:
Ok, let's increase it. and rename